shared-doc/attributes.adoc

```diff
@@ -19,13 +19,13 @@
 
 ifdef::EAPCDRelease[]
 // EAP CD release
-:productName: JBoss EAP Continuous Delivery
-:productNameFull: JBoss Enterprise Application Platform continuous delivery
+:productName: JBoss EAP CD
+:productNameFull: Red Hat JBoss Enterprise Application Platform Continuous Delivery
 :productVersion: 15
 :githubRepoUrl: https://github.com/jboss-developer/jboss-eap-quickstarts/tree/openshift/
 :jbossHomeName: EAP_HOME
 :DocInfoProductNameURL: jboss_enterprise_application_platform_continuous_delivery
-:DocInfoProductName: JBoss Enterprise Application Platform Continuous Delivery
+:DocInfoProductName: Red Hat JBoss Enterprise Application Platform Continuous Delivery
```

Comment:
I think the "Continuous Delivery" should be without the "Red Hat" prefix, but needs to be confirmed by the docs team.